src/app-layout/visual-refresh/tools.tsx

```diff
@@ -46,6 +46,7 @@ export default function Tools({ children }: ToolsProps) {
     toolsHide,
     toolsRefs,
     toolsWidth,
+    headerVariant,
```

Comment:
NIT, since in all places `headerVariant` is used it has a `headerVariant === 'high-contrast'` can we make `useAppLayoutInternals` hook return a `isHighContrast` header boolean value just for simplicity?

Reply:
Decided to keep the current implementation because it allows to re-use APpLayout typings